src/realizations/catchment/Bmi_Py_Formulation.cpp

```diff
@@ -48,29 +48,24 @@ std::string Bmi_Py_Formulation::get_formulation_type() {
     return "bmi_py";
 }
 
-string Bmi_Py_Formulation::get_output_line_for_timestep(int timestep, std::string delimiter) {
+std::string Bmi_Py_Formulation::get_output_line_for_timestep(int timestep, std::string delimiter) {
     // TODO: something must be added to store values if more than the current time step is wanted
     // TODO: if such a thing is added, it should probably be configurable to turn it off
     if (timestep != (next_time_step_index - 1)) {
-        throw std::invalid_argument("Only current time step valid when getting output for BMI Python formulation");
+        throw std::invalid_argument("Only current time step valid when getting output for BMI C++ formulation");
```

Comment:
shouldnt this still be `Python`?

Reply:
Yes. Austin. Just spotted it today. Thanks.